### PR TITLE
fix: fix permission issue in mobile

### PR DESCRIPTION
**Pull request by anselmwang**
On Android, Obsidian only has the permission to call `navigator.clipboard.readText()`. 

`const clipboardData = await navigator.clipboard.read();` will fail due to error "NotAllowedError: Read Permission Denied".

Also update dev dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
     "tslib": "^2.2.0",
     "tslint": "^6.1.3",
     "typescript": "^4.3.5",
-    "xmldom": "^0.6.0"
+    "xmldom": "^0.6.0",
+    "builtin-modules": "^3.2.0",
```
Comment (jglev) [inline package.json:24]: Thank you for drafting this to illustrate your point!

Would you be willing to say more about what build errors you were seeing without these being present?

Comment (anselmwang) [reply package.json:24]: I am not familiar with JS/TS development, so I might make stupid mistakes here.
```
$ node --version
v17.2.0
$ npm --version
8.1.4
```


Without "builtin-modules", here is my screenshot
image


Further without "yarn", here it is
image

Comment (jglev) [reply package.json:24]: Interesting, thank you! I could perhaps make it more explicit in the Readme that `yarn` is expected to be installed globally (as in its [main installation instructions](https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable)). I haven't seen the `builtin-modules` error before, but I will keep an eye out for it going forward, as well, and am certainly willing to add it here if it comes up again.

**Comment by jglev** [comment]
Thank you for this draft, @anselmwang! I've merged the main `2.2.0` branch just now, so this PR was automatically closed.

File: package.json (hunk 1 above)
Comment (anselmwang) [reply package.json:24]: You are totally right. The "builtin-modules" issue is because that I am running "npm run de" instead of "yarn dev". Thanks!

**Pull request closed without merging**